.github/workflows/S3deployment.yaml

```diff
@@ -0,0 +1,120 @@
+name: Deploy to AWS S3
```

Comment:
Could we name the file `cd`? 🤔

Reply:
I found it more intuitive, since it directly states where it is deployed to